### PR TITLE
Merge 2.5 onto master

### PR DESCRIPTION
**Pull request by soyuka**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -38,7 +45,7 @@
 
 * Allow to not declare GET item operation
 * Add support for the Accept-Patch header
-* Make the the `maximum_items_per_page` attribute consistent with other attributes controlling pagination 
+* Make the the `maximum_items_per_page` attribute consistent with other attributes controlling pagination
```
Comment (teohhanhui) [inline CHANGELOG.md]: the the

Comment (soyuka) [reply CHANGELOG.md]: needs to be fixed in 2.5 ? :p

Comment (teohhanhui) [reply CHANGELOG.md]: Yeah lol

**Comment by alanpoulain** [comment]
This PR is not good and causes conflicts. The SHA of the commits have changed.

**Pull request merged**